test/corpus/pluscal/assert.txt

```diff
@@ -0,0 +1,25 @@
+================================================================================
```

Comment:
I'm surprised these tests actually work; this isn't yet a documented tree-sitter feature, but using a suffix to the `===` and `---` like `|||` ensures you can disambiguate the TLA+ syntax from the tree-sitter test file syntax (this feature was actually added after encountering this problem).

Reply:
Yeah, they both run all the tests and detect errors

Reply:
All right, just for robustness reasons I think we should add the suffixes though.

Reply:
Fixed